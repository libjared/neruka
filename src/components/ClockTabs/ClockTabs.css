.ClockTabs {
  border: 1px solid #3c4043;
  border-radius: 8px;
}

.ClockTabs-nav {
  border-width: 0;
  border-bottom-width: 1px;
  border-style: solid;
  border-color: inherit;
}

.ClockTabs-nav > ul {
  list-style-type: none;
  margin: 0;
  padding: 0;
  display: flex;
}

.ClockTabs-nav > ul > li {
  display: inline-block;
  flex: 1 1 300px;
  line-height: 2em;
  text-align: center;
  line-height: 48px;
  text-transform: uppercase;
  font-size: 0.9em;
}
