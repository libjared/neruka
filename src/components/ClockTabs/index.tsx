function ClockTabs() {
  return null;
}

export default ClockTabs;
