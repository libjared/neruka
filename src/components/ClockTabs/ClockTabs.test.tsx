import { render } from "@testing-library/react";
import ClockTabs from ".";

it("renders", () => {
  const { asFragment } = render(<ClockTabs />);
  expect(asFragment()).toMatchSnapshot();
});
