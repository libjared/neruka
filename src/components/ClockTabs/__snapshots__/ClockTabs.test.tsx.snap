// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div
    class="ClockTabs"
  >
    <nav
      class="ClockTabs-nav"
    >
      <ul>
        <li>
          Timer
        </li>
        <li>
          Alarm
        </li>
      </ul>
    </nav>
    <div
      class="Timer"
    >
      <div
        class="Timer-display"
      >
        <div
          class="DurationInput-main"
          role="timer"
          tabindex="0"
        >
          <div
            class="DurationInput-area"
          >
            <span
              class="Digit lit"
            >
              1
            </span>
            <span
              class="Digit lit"
            >
              5
            </span>
            <span
              class="Label minutes lit"
            >
              m
            </span>
            <span
              class="Digit lit"
            >
              0
            </span>
            <span
              class="Digit lit"
            >
              0
            </span>
            <span
              class="Label seconds lit"
            >
              s
            </span>
          </div>
        </div>
      </div>
      <nav
        class="Timer-controls"
      >
        <input
          type="button"
          value="Start"
        />
        <input
          type="button"
          value="Reset"
        />
      </nav>
    </div>
  </div>
</DocumentFragment>
`;
